pkg/labels/labels.go

```diff
@@ -349,22 +349,22 @@ func (lbls Labels) GetModel() []string {
 // to.MergeLabels(from)
 // fmt.Printf("%+v\n", to)
 //   Labels{Label{key1, value3, source4}, Label{key2, value3, source4}}
-func (lbls Labels) MergeLabels(from Labels) {
+func (l Labels) MergeLabels(from Labels) {
 	fromCpy := from.DeepCopy()
 	for k, v := range fromCpy {
-		lbls[k] = v
+		l[k] = v
 	}
 }
 
 // SHA256Sum calculates lbls' internal SHA256Sum. For a particular set of labels is
```

Comment:
Can you change the function comment as well? ("lbls" -> "receiver's Labels `l`")